README.md

```diff
@@ -126,45 +119,22 @@ If you have a base path of `/api` and your path is defined as `hello`, then a re
 - [ ] Forwarding proxy with validation
 - [ ] Recording traffic to spec file
 - [ ] Data Persistence (turn Prism into a sandbox server)
-- [ ] Support files ending with `.yml` and extensionless files
-
-## Testing
-
-Prism has an extensive test suite. To run it, just use the regular `test` script
-
-```bash
-yarn test
-# or
-npm test
-```
+- [ ] Support files ending with `.yml` and extension-less files
 
-## Debugging
```

Comment:
I think the debugging section should stay or at least put a link in the relevant contributing section. Last place where I would look for technical help is the contributing file, what do you think @philsturgeon @tbarn

Reply:
I can add a link in another PR today.